lib/utilities/universal_functions.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:pokedex_async_redux/utilities/constants.dart';
+
+String capitalizeFirstLetter(String name) {
```

Comment:
instead of a function, make an extension instead.

Reply:
when you're done, put the extension inside the extensions.dart file inside your utilities folder